plugins/woocommerce/changelog/43227-as-diff-D129122

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Apply Rector suggestions for PHP 8.1
```

Comment:
For all of these diffs, we can probably leave out the reference to Rector, even though that was the tool used to identify the issues.  Something along the lines of "Improving return value type checks prior to usage."

Reply:
done https://github.com/woocommerce/woocommerce/pull/43227/commits/b56bfa1e537f9b8eecd13592faf6b61ddbdab93e